spec/system/visit_upload_page_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'UploadPage', type: :system do
+  let(:test_certificate) { 'ThisIsATestCertificate' }
+  it 'successfully submits a certificate' do
+    visit '/upload'
+    choose 'certificate_usage_signing', allow_label_click: true
+    fill_in 'certificate_value', with: test_certificate
+    click_button('Upload')
```

Comment:
really pedantic, but for sake of consistency, could this be `click_button 'Upload'` ? for `visit`, `choose` etc we don't use brackets

Reply:
This has been fixed with below commit
3fv3UwKG-system-test-for-upload-page-removed-brackets